### ISSUE TITLE
Overlay does not appear on embedded videos outside of YouTube.com

### ISSUE THREAD

**Issue by veritascitor** [meandavejustice/min-vid#1007]
Ever since the early August 2017 update (Min Vid 0.4.0), I've noticed that the Min Vid overlay does not appear on embedded videos outside of YouTube.com. For example, videos in RSS feeds shown on feedly.com, or videos embedded on any number of other sites. This used to work just fine, but now Min Vid seems to be limited to certain domains.

Firefox 55.0.2 on Windows 10, Min Vid 0,4,0, 0,4,2

Extensions:
Fire Gestures
Min Vid
SnoozeTabs
Social Fixer
Test Pilot
uBlock Origin

I've tried disabling various extensions to no effect. Is it possible some setting change has affected the way MinVid works? Is this a known issue? Is there more information I can provide in order to help solve it?

**Comment by SoftVision-CarmenFat** [comment]
Hi	@veritascitor and thanks for the report.

- I managed to reproduce the issue you described by accessing the https://feedly.com/i/welcome website and logging into my account.
The Min Vid overlay is **not** displayed on the Youtube videos shown in feeds.
This was tested on Windows 10 x64, with latest Fx release 55.0.2 and latest Min Vid v. 0.4.2.

- Also, I reproduced the initial behavior using an older sdk version of Min Vid (0.3.8 built in 3/7/2017) and the overlay was correctly displayed on the embedded Youtube videos.

@meandavejustice, seems that somewhere in the process of replacing Min Vid sdk to webextension, this feature has been lost. Is it possible to bring it back?
